account_tax_group_widget_base_amount/static/src/xml/tax_group.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-extend="AccountTaxGroupTemplate">
+        <t t-jquery=".oe_tax_group_name" t-operation="replace">
```

Comment:
Can you avoid a `replace` operation?

Reply:
Done. I wasn't sure I could use a t-jquery select to get t tags